hcl/parse.go

```diff
@@ -296,6 +322,41 @@ func loadResourcesHcl(list *ast.ObjectList) ([]*Resource, error) {
 			})
 		}
 
+		var config map[string]interface{}
+		if err := hcl.DecodeObject(&config, item.Val); err != nil {
+			log.Printf("HCLParse: unable to decode body: %v", err)
```

Comment:
For uniformity, let's keep all the hcl related log messages in the form: `hcl: Message`